lib/flight_scheduler/job_runner.rb

```diff
@@ -91,12 +91,18 @@ def run
       path = script_path.to_path
       FlightScheduler.app.job_registry.add(id, self)
 
+
       self.task = Async do |task|
         # Fork to create the child process [Non Blocking]
         self.child_pid = Kernel.fork do
           # Become the requested user and session leader
-          ENV['HOME'] = passwd.dir
-          ENV['USER'] = username
+          string_envs.merge!(
+            'HOME' => passwd.dir,
+            'LOGNAME' => username,
+            'PATH' => '/bin:/sbin:/usr/bin:/usr/sbin',
+            'USER' => username,
+            'flight_ROOT' => ENV['flight_ROOT'],
```

Comment:
I have a couple of reservations about hard coding `flight_ROOT` in like this. The `flight_ROOT` env var is a builder specific/integration specific detail. By hard coding it in, the application is starting to couple to a single deployment environment. Notionally the daemon should operate with `rvm` with no knowledge of flight ecosystem.

I acknowledge the fact that the other flight tools won't work without it (maybe?), so at this stage we can leave it. However eventually `flight_ROOT` should be a configurable env var or provided by `flight-scheduler-controller`.

Reply:
Having a mechanism for configuring what environment variables are preserved is definitely what we want down the road.  `sbatch` has the flags `--export`, `--export-file` and `--get-user-env` options to support this.  We may take that route.